docs/pages/database-access/guides/rds.mdx

```diff
@@ -18,99 +18,348 @@ This guide will help you to:
+by creating a join token.
+
+Generate a join token by running the following command on your workstation: 
+
+```code
+$ tctl tokens add --type=db
+```
+
+The next step depends on how you plan to run the Teleport Database Service:
+
+<Tabs>
+<TabItem label="Linux Server">
+
+Save the token in a file called `/tmp/token` on the host that will run the
+Database Service.
+
+</TabItem>
+<TabItem label="Kubernetes">
+
+Later in this guide, you will use this join token when configuring the Teleport
+Database Service.
+
+</TabItem>
+</Tabs>
 
 (!docs/pages/includes/database-access/alternative-methods-join.mdx!)
 
+### Prepare your environment
+
+Next, get your environment ready to run the Teleport Database Service:
+
+<Tabs>
+<TabItem label="Linux Host">
+
 Install Teleport on the host where you will run the Teleport Database Service:
 
 (!docs/pages/includes/install-linux.mdx!)
 
-Create the Database Service configuration:
+Generate a configuration for the Teleport Database Service:
 
-<ScopedBlock scope={["oss", "enterprise"]}>
+Run the command below on your Linux host. Assign <Var
+name="mytenant.teleport.sh:443" /> to the host **and port** of your Teleport
+Proxy Service or Enterprise Cloud/Team tenant. Assign <Var name="us-west-1" />
+to the region associated with the RDS databases you would like Teleport to
+discover.
 
 ```code
 $ teleport db configure create \
    -o file \
-   --proxy=teleport.example.com:3080 \
+   --proxy=<Var name="mytenant.teleport.sh:443" />  \
    --token=/tmp/token \
-   --rds-discovery=us-west-1
+   --rds-discovery=<Var name="us-west-1" />
 ```
 
-</ScopedBlock>
-<ScopedBlock scope={["cloud"]}>
+The command will generate a Database Service configuration with RDS/Aurora
+database auto-discovery enabled on the configured region and place it at the
+`/etc/teleport.yaml` location.
 
-```code
-$ teleport db configure create \
-   -o file \
-   --proxy=mytenant.teleport.sh:443 \
-   --token=/tmp/token \
-   --rds-discovery=us-west-1
-```
 
-</ScopedBlock>
+</TabItem>
+<TabItem label="Kubernetes Cluster">
 
-The command will generate a Database Service configuration with RDS/Aurora
-database auto-discovery enabled on the `us-west-1` region and place it at the
-`/etc/teleport.yaml` location.
+(!docs/pages/kubernetes-access/helm/includes/helm-repo-add.mdx!)
+
+</TabItem>
+</Tabs>
 
-## Step 3/6. Create an IAM policy for Teleport
+## Step 3/6. Create IAM policies for Teleport
 
-Teleport needs AWS IAM permissions to be able to:
+The Teleport Database Service needs AWS IAM permissions to be able to:
 
 - Discover and register RDS instances and Aurora clusters.
 - Configure IAM authentication for them.
 
-Before you can generate IAM permissions, you must provide the Teleport Database
-Service access to AWS credentials.
+In this step, we will show you how to provide the Teleport Database Service
+access to AWS credentials by using the Teleport configuration file we created
+earlier to automatically create and attach IAM policies:
```

Comment:
Isn't the config file only created when following the "Linux Host" tabs? Wouldn't that make this line confusing to those on the K8S tabs?

Reply:
The second half of this paragraph is a holdover from instructions I added to use `teleport db configure bootstrap` and then deleted. I've made this paragraph more general.